101-bundle-json.md

```diff
@@ -46,7 +46,7 @@ The following is an example of a `bundle.json` for a bundled distributed as a _t
             "image": "technosophos/microservice:1.2.3",
             "description": "my microservice",
             "digest": "sha256:aaaaaaaaaaaa...",
-            "uri": "urn:image1uri",
+            "image": "image1name:image1tag",
```

Comment:
This would result in duplicate keys for `image`.

Reply:
oh! I missed that. Good catch.